_data/events.yml

```diff
@@ -76,6 +76,7 @@
   linkedIn: "https://www.linkedin.com/posts/itenium_itenium-keeponlearning-codecomplete-activity-7200798368653094912-f3ac?utm_source=share&utm_medium=member_desktop"
   instagram: "https://www.instagram.com/p/C7d8Gy1sfrH/?utm_source=ig_web_copy_link&igsh=MzRlODBiNWFlZA=="
   twitter: "https://x.com/itenium_be/status/1801544035573579999"
+  githubURl: "https://github.com/itenium-be/CodeComplete/blob/master/CodeComplete-Part2.pptx"
```

Comment:
Are you sure the name is "githubURL"?
Also the url is: https://github.com/itenium-be/CodeComplete
Without the rest of it

Reply:
I checked, it's not
The correct name is "githubUrl"